### PR TITLE
enhancement: redesigned Navbar and Footer

### PR DESCRIPTION
**Pull request by dhairyathedev**
- Redesigned the navbar
- Redesign the footer

### CODE REVIEW

**Comment by dhairyathedev** [comment]
@ashutosh-rath02 here's the changes in the UI.

Demonstration Video: https://youtu.be/3UvRmTUcETQ

**Comment by ashutosh-rath02** [comment]
Hey @dhairyathedev awesome work there, just the logo on the footer and navbar needs to be changed. Keep it as it is now

**Comment by dhairyathedev** [comment]
> Hey @dhairyathedev awesome work there, just the logo on the footer and navbar needs to be changed. Keep it as it is now


Hey @ashutosh-rath02, you mean to keep the current logo with black color, right?
Screenshot 2024-05-12 at 10 04 41 AM

**Comment by dhairyathedev** [comment]
Hey @ashutosh-rath02 , reverted to old logo.
<br />
image

**Comment by dhairyathedev** [comment]
Please review and merge the PR.

File: components/AuthButton.tsx [hunk 1]
```diff
@@ -33,19 +41,21 @@ export default function AuthButton({ user, height, width }: AuthButtonProps) {
 
   return user ? (
     <Button
-      variant="outline"
-      className="inline-flex h-10 animate-shimmer items-center justify-center rounded-md border border-slate-800 bg-[linear-gradient(110deg,#000103,45%,#1e2631,55%,#000103)] bg-[length:200%_100%] px-12 font-medium text-white text-lg transition-colors focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50"
+      variant="newDefault"
+      className={`animate-shimmer h-10 ${className}`} // Add className prop to the Button
```
Comment (ashutosh-rath02) [inline components/AuthButton.tsx:45]: Remove comments

File: components/footer.tsx [hunk 2]
```diff
@@ -13,42 +18,104 @@ export default function Footer() {
...
-          <span className="text-lg font-black flex flex-row items-center justify-center gap-2">
+    <footer className="max-w-screen-2xl mx-auto m-2 p-4 w-full flex flex-col space-y-12 sm:flex-row justify-between items-start">
+      <div className="flex flex-col max-w-96">
+        <div className="flex items-center gap-x-2">
+          {/* <div>
+            <IconBrandGithubFilled className="text-primaryLight sm:w-[36px] sm:h-[36px] w-[30px] h-[30px]" />
+          </div>
+          <h1 className="sm:text-3xl text-2xl font-semibold text-primaryLight">
+            git-re
+          </h1> */}
```
Comment (ashutosh-rath02) [inline components/footer.tsx:44]: remove unwanted comments

File: components/footer.tsx [hunk 3]
```diff
@@ -13,42 +18,104 @@ export default function Footer() {
...
+          target="_blank"
+          rel="noopener noreferrer"
+          className="w-48 mt-8 rounded-sm py-4 text-base font-semibold dark:text-white flex flex-row items-center bg-primary text-white justify-center"
+        >
+          <IconStarFilled className="mr-2" />
+          Star on GitHub
         </Link>
-
-        <p className="text-xs font-light">
-          &copy; {currentYear} Made with 🤯 by Ashutosh Rath
```
Comment (ashutosh-rath02) [inline components/footer.tsx:41]: Keep this line

File: components/footer.tsx [hunk 4]
```diff
@@ -13,42 +18,104 @@ export default function Footer() {
...
+            >
+              Privacy Policy
+            </Link>
+            <Link
+              href="/terms"
+              className="text-[#7E7E7E] font-medium dark:text-white"
+            >
+              Terms of Service
+            </Link>
+          </div>
```
Comment (ashutosh-rath02) [inline components/footer.tsx:83]: We don't have these sections. Please remove them

File: components/navbar.tsx [hunk 5]
```diff
@@ -14,27 +15,40 @@ export default async function Navbar() {
...
-        <a
-          href={repositoryUrl}
+    <nav className="max-w-screen-2xl mx-auto m-2 mt-7 p-4 w-full flex items-center justify-between">
+      <div className="flex items-center gap-x-2">
+        {/* <div>
+          <IconBrandGithubFilled className="text-primaryLight sm:w-[36px] sm:h-[36px] w-[30px] h-[30px]" />
+        </div>
+        <h1 className="sm:text-3xl text-2xl font-semibold text-primaryLight">
+          git-re
+        </h1> */}
```
Comment (ashutosh-rath02) [inline components/navbar.tsx:25]: Remove comments

File: components/navbar.tsx [hunk 6]
```diff
@@ -14,27 +15,40 @@ export default async function Navbar() {
...
+          git-re
+        </Link>
+      </div>
+      <div className="flex flex-row items-center space-x-8">
+        <Link
+          href="https://github.com/ashutosh-rath02/git-re"
           target="_blank"
           rel="noopener noreferrer"
-          className="text-lg hover:text-primary cursor-pointer flex items-center gap-1"
+          className="text-[3C3C3C] hover:text-primary font-semibold transition sm:block hidden"
```
Comment (ashutosh-rath02) [inline components/navbar.tsx:39]: Put the color in `tailwind.config.js` and use from there

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

**Comment by ashutosh-rath02** [comment]
image
Hey @dhairyathedev the gaps on both side is too small which is looking odd, keep the gaps use `max-w-` to adjust the width. Check the present website to see gaps on both sides, 
Do the same for footer.

**Comment by ashutosh-rath02** [comment]
@dhairyathedev pull the recent changes once then work on it... as some changes have been introduced, I want to check how it looks with the whole website.

**Comment by dhairyathedev** [comment]
> image Hey @dhairyathedev the gaps on both side is too small which is looking odd, keep the gaps use `max-w-` to adjust the width. Check the present website to see gaps on both sides, Do the same for footer.

I have kept the same width as the current site, and the padding and margin are proper on my device with proper gaps on each side.

Screenshot 2024-05-12 at 10 50 41 AM

**Comment by ashutosh-rath02** [comment]
@dhairyathedev You can check the preview

**Comment by dhairyathedev** [comment]
> @dhairyathedev You can check the preview https://git-re-git-fork-dhairyathedev-main-lucif3rins-projects.vercel.app/ Check for larger screens maybe

I need access required to view the preview deployment. I have sent access request, please approve. @ashutosh-rath02

**Comment by ashutosh-rath02** [comment]
Done

**Comment by dhairyathedev** [comment]
> Done

@ashutosh-rath02 
Screenshot 2024-05-12 at 11 17 38 AM

**Comment by dhairyathedev** [comment]
> image Hey @dhairyathedev the gaps on both side is too small which is looking odd, keep the gaps use `max-w-` to adjust the width. Check the present website to see gaps on both sides, Do the same for footer.

I have updated the mentioned changes and the gap issue. Please check if it still exists.

**Comment by ashutosh-rath02** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Comments	Updated (UTC)
> **git-re**	🔄 Building ([Inspect](https://vercel.com/lucif3rins-projects/git-re/3innipjvTkktk11xR5dScQgvXNpW))	[Visit Preview](https://vercel.live/open-feedback/git-re-git-fork-dhairyathedev-main-lucif3rins-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)	💬 [**Add feedback**](https://vercel.live/open-feedback/git-re-git-fork-dhairyathedev-main-lucif3rins-projects.vercel.app?via=pr-comment-feedback-link)	May 12, 2024 5:52am

@dhairyathedev Can you check the review now?

**Comment by dhairyathedev** [comment]
> > **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> > Name	Status	Preview	Comments	Updated (UTC)
> > **git-re**	🔄 Building ([Inspect](https://vercel.com/lucif3rins-projects/git-re/3innipjvTkktk11xR5dScQgvXNpW))	[Visit Preview](https://vercel.live/open-feedback/git-re-git-fork-dhairyathedev-main-lucif3rins-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)	💬 [**Add feedback**](https://vercel.live/open-feedback/git-re-git-fork-dhairyathedev-main-lucif3rins-projects.vercel.app?via=pr-comment-feedback-link)	May 12, 2024 5:52am
> 
> @dhairyathedev Can you check the review now?

Yes I can see the preview

**Comment by ashutosh-rath02** [comment]
Let's connect over discord https://discordapp.com/users/519818032973873152

**Pull request merged**